source/gui/settingsDialogs.py

```diff
@@ -517,8 +517,8 @@ def makeSettings(self, settingsSizer):
 		# Translators: This is a label for a setting in voice settings (an edit box to change voice pitch for capital letters; the higher the value, the pitch will be higher).
 		capPitchChangeLabel=wx.StaticText(self,-1,label=_("Capital pitch change percentage"))
 		settingsSizer.Add(capPitchChangeLabel)
-		self.capPitchChangeEdit=wx.TextCtrl(self,wx.NewId())
-		self.capPitchChangeEdit.SetValue(str(config.conf["speech"][getSynth().name]["capPitchChange"]))
+		self.capPitchChangeEdit=wx.SpinCtrl(self, min=0, max=100,

```

Comment:
The minimum here is -100, not 0. (The value is relative, not multiplicative.) Do these controls support negative values?